IPIP/0001-lightweight-rfc-process.md

```diff
@@ -0,0 +1,142 @@
+# IPIP 0001: Lightweight "RFC" Process for IPFS Specifications
+
+- Start Date: 2022-06-10
+- Related Issues:
+  - [ipfs/specs/issues/286](https://github.com/ipfs/specs/issues/286)
+
+# Summary
+
+This _InterPlanetary Improvement Proposal_ (IPIP) introduces a lightweight
+"request for comments/change" process for the IPFS specifications
+[repository][1].
+
+[1]: https://github.com/ipfs/specs/
+
+# Motivation
+
+Today, protocol design discussions often take place in a repository of an IPFS
+implementation. These conversations are unintentionally obscured from the useful input of [Specs Stewards], other
+implementations, service operators and the wider IPFS Community.
+
+The IPFS Project needs a mechanism for proposing and evaluating specification
+improvements that are not tied to a specific programming language
+or implementation of IPFS.
+
+# Detailed design
+
+Adopt an informal IPIP process for the [ipfs/specs][1] repository, providing a
+minimal structure for opening, reviewing, and merging specification changes.
+
+## Opening an improvement proposal (IPIP)
+
+Changes to IPFS specifications can be proposed by opening a Git pull-request
+(PR) against the repository. In addition to specification changes, such PR must
+include a short IPIP document based on the template in `IPIP/0000-template.md`.
```

Comment:
@lidel is there also a "may" in here around changing the spec as well? IIUC I would expect a change (or the initial) webdav gateway proposal would generally include both a `IPIP/000N-webdav-gateway.md` that's minimal and more practically a change to `specs/WEBDAV_GATEWAY.md`.

To that end do we have a spec template that we want people to use in laying out their specs so that there's some kind of uniformity?

Reply:
Having template for specs is a good idea. 

- [ ] Note to self:  do basic repo cleanup, add spec template